kolibri/core/assets/src/views/AttemptLogList.vue

```diff
@@ -230,6 +241,7 @@
         handleSectionChange,
         handleQuestionChange,
         displaySectionTitle,
+        questionNumberLabel$,
```

Comment:
Noting this here -- but we're labeling the selected question "Question <selected question number>" but then the list of questions is listed from `1 - N` - should we be calling the questions in this case always by their position in the greater quiz?

![image](https://github.com/learningequality/kolibri/assets/6356129/76049a28-a2b8-4936-aca4-60319041ed3b)